policyengine_uk/variables/gov/dfe/extended_childcare_entitlement/extended_childcare_entitlement_eligible.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_uk.model_api import *
+
+
+class extended_childcare_entitlement_eligible(Variable):
+    value_type = bool
+    entity = BenUnit
+    label = "Whether this benefit unit is eligible for extended childcare entitlement"
+    documentation = "Checks if the benefit unit meets all eligibility conditions (work and income requirements) for extended childcare entitlement"
+    definition_period = YEAR
+
+    def formula(benunit, period, parameters):
+        # Check income condition - must be true for all family members (except children)
+        meets_income_condition = benunit.all(
```

Comment:
Then use `person` inside
```suggestion
        person = benunit.members
        person_meets_income_condition = person("extended_childcare_entitlement_meets_income_requirements",
                period,
            )
            | person("is_child", period)
        meets_income_condition = benunit.all(person_meets_income_condition)
```